distribution/docker/build.gradle

```diff
@@ -186,3 +186,37 @@ assemble.dependsOn "buildDockerImage"
 if (tasks.findByName("composePull")) {
     tasks.composePull.enabled = false 
 }
+
+/*
+ * The export subprojects write out the generated Docker images to disk, so
+ * that they can be easily reloaded, for example into a VM.
+ */
+subprojects { Project subProject ->
+  if (subProject.name.contains('docker-export')) {
+    apply plugin: 'distribution'
+
+    def oss = subProject.name.startsWith('oss')
```

Comment:
Please use static types. It makes the code easier to reason about, eg String here.